docker/prod/Dockerfile

```diff
@@ -15,8 +15,8 @@ ENV DOCKER=1
 ENV APP_USER=app
 ENV APP_PATH=/app
 ENV APP_DATA_PATH=/var/openproject/assets
-ENV PGVERSION="13"
-ENV PGVERSION_CHOICES="13 15 17"
+ENV PGVERSION="17"
```

Comment:
This is not correct, as changing this line will not auto-migrate for existing installations. If you install OpenProject from a new installation, it's preferable to set PGVERSION=17 explicitly